plugin/src/main/java/at/helpch/chatchat/config/holder/MessagesHolder.java

```diff
@@ -36,6 +37,8 @@ public final class MessagesHolder {
     private Component unignoredPlayer = text("Successfully un-ignored <player>.", GREEN);
     private Component alreadyIgnored = text("You are already ignoring <player>.", RED);
     private Component notIgnored = text("You are not ignoring <player>.", RED);
+    private Component notIgnoringAnyone = text("You are not ignoring anyone.", RED);
+    private Component ignoredPlayerList = text("You are ignoring following players: <ignoredPlayers>.", YELLOW);
```

Comment:
Change from `<ignoredPlayers>` to `<ignored_players>`.

Also, you can append multiple components. Maybe make the text yellow and the list white by default?

Also, since it is representing a list, the variable should probably be named `ignoredPlayersList`